qt/python/mantidqtinterfaces/mantidqtinterfaces/Engineering/gui/engineering_diffraction/tabs/gsas2/call_G2sc.py

```diff
@@ -61,7 +66,7 @@ def add_pawley_reflections(pawley_reflections, project, d_min):
         phase.data["Pawley ref"] = gsas_reflections
 
 
-def set_max_number_cycles(number_cycles):
+def set_max_number_cycles(gsas_project, number_cycles):
```

Comment:
Does this fix a bug? If so could be due a release note?

Reply:
No, it's not a bug, I changed it to be consistent with the other functions and so I can write a test for it with mocking the project